frontend/common/src/components/UserProfile/ExperienceAccordion/individualExperienceAccordions/AwardAccordion.tsx

```diff
@@ -40,6 +40,7 @@ const AwardAccordion: React.FunctionComponent<AwardAccordionProps> = ({
               {skill.name?.[locale]}
               <br />
               {skill.description?.[locale]}
+              {skill.experienceSkillRecord?.details}
```

Comment:
This may be a good idea to follow the pattern of the other accordions and wrap each in their own `<p>` tag:

```tsx
        <ul key={index}>
          <li>
            <p data-h2-font-color="b(lightpurple)">{skill.name?.[locale]}</p>
            <p>{skill.description?.[locale]}</p>
            <p>{skill.experienceSkillRecord?.details}</p>
          </li>
        </ul>
```

Reply:
[here](https://github.com/GCTC-NTGC/gc-digital-talent/pull/3460/commits/080e3b2406c2029f5345e414d52174455722350d)